src/background.ts

```diff
@@ -138,6 +139,22 @@ protocol.registerSchemesAsPrivileged([
 
 const firstUrl = process.env.VITE_DEV_SERVER_URL ?? "app://./index.html";
 
+const tempProjectPath = path.join(fixedUserDataDir, "project.tmp.json");
+
+const setTempProject = async (tempProject: WorkspaceType) => {
```

Comment:
こちらとかが混じってそうだなと･･･！
関数名も含めて整理すると･･･こう？
```suggestion
const saveWorkspace = async (workspace: WorkspaceType) => {
```